### DISCUSSION TITLE
when using stream if host fails entire app crashes

### DISCUSSION THREAD

**Discussion by formytech** [ZoeyVid/NPMplus#729] [Q&A] [answered]
is there anyway to prevent this, if a stream host stops responding, it brings the entire container down and it wont start up again unless manually removed from config or host comes online

**Comment by Zoey2936** [comment]
can you provide your docker logs?

**Comment by formytech** [comment]
i will have to put the streams back, but basically if one of the origin streams stops responding, the entire npmplus crashes and tosses a error about the stream config file, and will not come back up unless the origin does, or manually delete the config file and that is a pita

**Reply by Zoey2936** [reply, marked as answer]
can you provide the logs?
